researchstack-sdk/src/main/java/org/sagebionetworks/bridge/researchstack/BridgeDataProvider.java

```diff
@@ -47,27 +47,23 @@
 public abstract class BridgeDataProvider extends DataProvider {
     private static final Logger logger = LoggerFactory.getLogger(BridgeDataProvider.class);
 
-    protected final Gson gson = RestUtils.GSON;
-
     // set in initialize
-    protected UserLocalStorage userLocalStorage;
-    protected ConsentLocalStorage consentLocalStorage;
     protected TaskHelper taskHelper;
     protected UploadHandler uploadHandler;
 
+    private final StorageAccessWrapper storageAccessWrapper;
```

Comment:
What's with StorageAccessWrapper?  The header says it is to help unit tests, but I think it just creates another layer of abstract confusion, especially since StorageAccess is a singleton.  The point of a singleton is there is only one access point for the class, but then we pass it around in a wrapper?  Why not just pass around StorageAccess at that point?